Src/Base/AMReX_Math.H

```diff
@@ -57,6 +57,32 @@ using std::isinf;
 
 #endif
 
+AMREX_GPU_HOST_DEVICE inline
+void sincos (double x, double* sinx, double* cosx)
```

Comment:
Suggestion: let's make this a more C++-ish interface and return a pair or tuple?

Otherwise, we should restrict the pointers.

Reply:
Sure, we can return a pair.